funcx_sdk/tests/unit/test_client.py

```diff
@@ -147,3 +148,75 @@ def test_pending_tasks_always_fetched():
         if should_fetch:
             args, _ = fxc.web_client.get_task.call_args
             assert sf == args[0]
+
+
+def test_build_container(mocker, login_manager):
+    mock_data = mocker.Mock()
+    mock_data.data = {"container_id": "123-456"}
+    login_manager.get_funcx_web_client.post = mocker.Mock(return_value=mock_data)
+    fxc = funcx.FuncXClient(do_version_check=False, login_manager=login_manager)
+    spec = ContainerSpec(
+        name="MyContainer",
+        pip=[
+            "matplotlib==3.5.1",
+            "numpy==1.18.5",
+        ],
+        # payload_url="https://raw.githubusercontent.com/funcx-faas/funcx-container-service/dev/README.md",  # noqa E501
+        payload_url="https://github.com/funcx-faas/funcx-container-service.git",
```

Comment:
I take it the README is a clue to the dev?  Suggest saying that more directly, else this URL may get lost in the future.  Something like:
```python
# See https://raw.githubusercontent.com/funcx-faas/funcx-container-service/dev/README.md
# for some background information on how to test.
```

Reply:
No the README is not documentation on how to test. I was looking for a simple file to test the upload with. This is no longer supported in any event, so the test just points to a GitHub repo